lib/tmbundle.rb

```diff
@@ -56,7 +56,11 @@ def install name
     require 'tmbundle/bundle_name'
     name = BundleName.new(name)
     install_path = bundles_dir.join(name.install_name).to_s
-    system('git', 'clone', name.git_url, install_path)
+    success = system('git', 'clone', name.git_url, install_path)
+    if not success
```

Comment:
`unless` would be more idiomatic here, but I'm ok with merging it as it is, let me know if you wanna fix it so that I'll wait for the update